src/components/templates/Jazzbar/JazzBar/JazzBar.tsx

```diff
@@ -1,161 +1,62 @@
-
-  const { hide: enableUserAudio, show: disableUserAudio } = useShowHide(
-    isReactionsMuted
-  );
-
-  useEffect(() => {
-    if (isReactionsMuted) {
-      disableUserAudio();
-    } else {
-      enableUserAudio();
-    }
-  }, [isReactionsMuted, disableUserAudio, enableUserAudio]);
-
   useEffect(() => {
     analytics.trackEnterJazzBarEvent();
   }, [analytics]);
 
-  useEffect(() => {
-    seatedAtTable && analytics.trackSelectTableEvent();
-  }, [analytics, seatedAtTable]);
-
-  const { joinHuddle, inHuddle } = useVideoHuddle();
-
-  const joinTable = useCallback(
-    (table) => {
-      // @debt TOOD Fix this
-      joinHuddle(userId || "", `${venue.id}-${table}`);
-      setSeatedAtTable(table);
-    },
-    [joinHuddle, userId, venue.id]
-  );
-
-  const leaveTable = useCallback(async () => {
-    if (!userId) return;
-
-    await unsetTableSeat(userId, { venueId: venue.id });
-    setSeatedAtTable(undefined);
-  }, [userId, venue.id]);
-
-  if (!inHuddle && seatedAtTable) {
-    leaveTable();
+  if (!userId) {
+    return <Loading />;
   }
 
-  if (!venue) return <>Loading...</>;
-
   return (
     <>
-      {/*
-      <VenueWithOverlay
-        venue={venue}
-        containerClassNames={`music-bar ${containerClasses}`}
-      >
-      */}
       {parentVenue && <BackButton variant="simple" space={parentVenue} />}
 
       {!venue.hideVideo && (
-        <div className="component-media-object">
-          <div className="iframe-container">
-            {embedIframeUrl ? (
-              <iframe
-                key="main-event"
-                title="main event"
-                className="iframe-video"
-                src={embedIframeUrl}
-                frameBorder="0"
-                allow={IFRAME_ALLOW}
-              />
-            ) : (
-              <div className="iframe-video">
-                Embedded Video URL not yet set up
-              </div>
-            )}
-          </div>
-          <div className="media-controls"></div>
-        </div>
+        <MediaPlayer url={venue.iframeUrl} autoPlay={venue.autoPlay || false} />
       )}
 
-      {/*!seatedAtTable && (
-        <UserList
-          usersSample={venue.recentUsersSample ?? ALWAYS_EMPTY_ARRAY}
-          userCount={venue.recentUserCount ?? 0}
-          activity={venue.activity ?? "here"}
-        />
-      )*/}
-
-      {/*seatedAtTable && (
-        <TableHeader
-          seatedAtTable={seatedAtTable}
-          setSeatedAtTable={setSeatedAtTable}
-          venueName={venue.name}
-          tables={jazzbarTables}
-          venueId={venue.id}
-          defaultTables={JAZZBAR_TABLES}
-        />
-      )*/}
+      {/* TODO Does this want extracting out into its own component? */}
+      <div className={styles.componentSpaceInfo}>
+        {venue.name && <h1>{venue.name}</h1>}
+      </div>
+
       {venue.description?.text && (
         <div className="row">
           <div className="col">
```

Comment:
[code smell] These look like depending on a global CSS file. Should be namespaced or removed.

```suggestion
        <div className="row">
          <div className="col">
```